FirebaseRemoteConfig/Sources/FIRRemoteConfigComponent.m

```diff
@@ -107,4 +115,15 @@ + (void)load {
   return @[ rcProvider ];
 }
 
+- (void)rolloutsStateDidUpdate:(nonnull FIRRolloutsState *)rolloutsState {
+  // calling the subscriber to update the change
+  [self.subscribers[@"nameSpace"] onRolloutsStateChanged:rolloutsState];
+}
+
+- (void)registerRolloutsStateSubscriber:(nonnull NSString *)nameSpace
+                             subscriber:(nullable id<FIRRolloutsStateSubscriber>)subscriber {
+  // adding the registered subscriber reference
+  self.subscribers[nameSpace] = subscriber;
+}
+
```

Comment:
For the `subscribers` type, I think we want a 1 to many namespace to subscribers. Eg. Dictionary<NSString *, NSArray<Subscriber>> so that we can have multiple subscribers per namespace

Reply:
To support future SDKs like AQS